static/js/containers/VideoDetailPage.js

```diff
@@ -53,29 +56,40 @@ class VideoDetailPage extends React.Component {
     }
   };
 
-  openDialog = () => {
+  openEditDialog = () => {
     const { dispatch, video, editable } = this.props;
     if (video && editable) {
       dispatch(setTitle(video.title));
       dispatch(setDescription(video.description));
     }
-    dispatch(setDialogVisibility(true));
+    dispatch(setEditDialogVisibility(true));
+  };
+
+  openshareDialog = () => {
+    const {dispatch} = this.props;
+    dispatch(setShareDialogVisibility(true));
   };
 
   submitForm = () => {
     const { dispatch, videoKey, videoDetailUi } = this.props;
-    const { title, description } = videoDetailUi.dialog;
+    const { title, description } = videoDetailUi.editDialog;
```

Comment:
I see this was like this before, but this is a little strange, we can do this instead to do the nested destructuring:

```js
const { editDialog: { title, description }} = videoDetailUi;
```